docs/src/components/prototyper/download-button.jsx

```diff
@@ -1,20 +0,0 @@
-import React from 'react';
```

Comment:
I am not sure I follow the logic of adding this to the theme, instead of having it in `src/components/`. To me, what is in the theme is usable by different pages. In case of the prototyper, it's not the case